### PR TITLE
Make changes models

### PR DESCRIPTION
**Pull request by jpmaHub**
Modify models to align with schema feedback

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -69,84 +33,10 @@
...
-  create_table "rpi_fixed_data", force: :cascade do |t|
-    t.string "year"
-    t.decimal "rpi"
-    t.decimal "real"
-    t.decimal "fr_uplift"
-    t.decimal "rc_factor"
-    t.decimal "rc_uplift"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.string "removed"
```
Comment (rrpff) [inline db/schema.rb]: Suggest we make `removed` a boolean value, as there will only be two cases (removed or not) and it's faster to query.

**Review by rrpff — APPROVED** [review]
The changes look really good - just one small comment in there 👍

**Comment by rf-mt** [comment]
just thinking - should year be an integer? might be tricky trying to add 1 to a year that's a string? @zuren @jpmaHub  thoughts?

**Comment by rf-mt** [comment]
In the schema annotations yday @zuren suggested that` property_type` should be an enum. Jaseera and I were thinking this originally (and for other fields such as `mra_archetype`)

**Comment by rrpff** [comment]
Agree r.e. `year` being an int, and an enum being used for `property_type`. Feel `mra_archetype` looks variable enough that hardcoding it in an enum might be a problem? i.e. they might be adding archetypes more frequently than new property types (which may never change.)

File: app/models/rent_charge.rb [hunk 2]
```diff
@@ -1,4 +1,6 @@
 class RentCharge < ApplicationRecord
+  
+  enum property_type: [:BUN, :FLT, :HOS, :HSE, :MSN]
```
Comment (rrpff) [inline app/models/rent_charge.rb]: Do we know what these mean? Would recommend human-readable enum values over shortened codes. Don't get much value out of the latter, and the former will be more self-documenting.

Assuming bungalow, flat, hostel, house... not sure about MSN? Maisonette?

Comment (jpmaHub) [reply app/models/rent_charge.rb]: Yes. think so

File: db/schema.rb [hunk 3]
```diff
@@ -10,143 +10,32 @@
...
     t.string "comments"
     t.decimal "rr_count"
-    t.string "property_type"
-    t.decimal "base_data_bed_size"
+    t.integer "property_type"
+    t.integer "base_data_bed_size"
     t.decimal "bedroom_weight"
     t.string "mra_archetype"
     t.decimal "jan_1999_asset_values"
     t.decimal "formula_rent_prev_year"
```
Comment (rf-mt) [inline db/schema.rb]: do we need `formula_rent_prev_year` and `rent_cap_prev_year` now that the table will contain rent details for all years? Also, I know a lot of the rent details can be calculated dynamically but don't we need to persist at least the current rent charge to the db? To me it makes sense to persist `formula_rent_this_year`, `rent_cap_this_year`, and  `uprated_actual`, and everything else can be calculated dynamically.

Comment (rrpff) [reply db/schema.rb]: Agree with this 100%

**Review by rf-mt — APPROVED** [review]

**Pull request merged**